cg/services/pre_analysis_quality_check/quality_controller/utils.py

```diff
@@ -0,0 +1,46 @@
+from typing import Callable
+
+from cg.services.pre_analysis_quality_check.quality_controller.quality_controller import (
+    QualityController,
+)
+from cg.constants.sequencing_quality_constants import (
+    CASE_SEQUENCING_QUALITY_CHECKS,
+    SAMPLE_SEQUENCING_QUALITY_CHECKS,
+)
+from cg.store.models import Case, Sample
+
+
+def get_sequencing_quality_checks_for_case(
+    data_analysis: str,
+) -> list[Callable] | None:
+    sequencing_quality_checks: list[Callable] | None = CASE_SEQUENCING_QUALITY_CHECKS.get(
+        data_analysis, None
+    )
+    if not sequencing_quality_checks:
+        raise NotImplementedError(
+            f"No sequencing quality check have implemented for workflow {data_analysis}."
+        )
+    return sequencing_quality_checks
+
+
+def run_pre_analysis_quality_check(
+    obj: Case | Sample,
+    quality_checks: list[Callable],
+) -> bool:
+    return QualityController.run_qc(obj, quality_checks)
+
+
+def run_case_pre_analysis_quality_check(
+    case: Case,
+) -> bool:
+    sequencing_quality_checks: list[Callable] = get_sequencing_quality_checks_for_case(
+        case.data_analysis
+    )
+
+    quality_checks: list[Callable] = sequencing_quality_checks
+    return run_pre_analysis_quality_check(obj=case, quality_checks=quality_checks)
```

Comment:
```suggestion
    return QualityController.run_qc(case, quality_checks)
```
and names parameters

Reply:
yesh